Jenkinsfile_develop_osx

```diff
@@ -1,6 +1,6 @@
 //Jenkins pipelines are stored in shared libaries. Please see: https://github.com/NREL/cbci_jenkins_libs
 
-@Library('cbci_shared_libs') _
+@Library('cbci_shared_libs@geometry_improvements') _
```

Comment:
This is just me saying to jenkins that it should use the branch I created where I re-enabled the degenerate intersection tests (which now pass)